src/auth/axios.tsx

```diff
@@ -20,7 +20,7 @@ export const requestInterceptor: (
 
 Instance.interceptors.request.use(requestInterceptor);
 
-const INVALID_ACCESS_TOKEN: string = 'Given access token is expired or invalid';
+const INVALID_ACCESS_TOKEN = 'Given access token is expired or invalid';
```

Comment:
Prettier told me to get rid of it 🤷‍♂️

Reply:
Wait really? What was the error message?

Reply:
Something along the lines of the string type hint being unnecessary because the constant could only be a string / was obviously a string. I don't remember the exact text, unfortunately.